save-agent/src/linuxX64Main/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -70,51 +74,53 @@ class SaveAgent(internal val config: AgentConfiguration,
      * @return Unit
      */
     suspend fun start() = coroutineScope {
-        logInfo("Starting agent")
+        logInfoCustom("Starting agent")
```

Comment:
Do we have some additional information at this moment? Simple message looks little bit senselessly